lnrpc/rpc.proto

```diff
@@ -1051,11 +1051,14 @@ message GetInfoResponse {
     /// Number of active channels
     uint32 num_active_channels = 4 [json_name = "num_active_channels"];
 
+    /// Number of inactive channels
+    uint32 num_inactive_channels = 5 [json_name = "num_inactive_channels"];
```

Comment:
Proto definitions must keep their numbers for backwards compatibility. Instead `num_inactive_channels` should be defined with the first non-used number (looks like it is 15).